packages/cactus-plugin-htlc-eth-besu-erc20/src/main/typescript/environment.ts

```diff
@@ -0,0 +1,7 @@
+//TODO: Use this parameters with the plugin vault
```

Comment:
@jordigiam In response to your `TODO` comment, yes, please do that, we cannot hard-code private keys, contract addresses, account addresses in the code for non-test-code.

Reply:
Done